src/cms/database/migrations/2023.06.26T00.00.06.access-status.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
```

Comment:
@ayumi-oxd 
The values in this file come from confluence, but the requirements aren't 100% finalized yet (see ticket CM-652 and this page in confluence:  https://apps.nrs.gov.bc.ca/int/confluence/display/BCPRS/Access-status? )

I used the draft version for dev purposes, but it will probably need to be updated while I am on vacation and before this gets merged to develop.